xfields/beam_elements/beambeam_src/beambeam3d.h

```diff
@@ -24,14 +24,12 @@ void do_luminosity(BeamBeamBiGaussian3DData el, LocalParticle *part,
     // init record table
     BeamBeamBiGaussian3DRecordData lumi_record = NULL;
     LumiTableData lumi_table                   = NULL;
-    RecordIndex lumi_table_index               = NULL;
     lumi_record = BeamBeamBiGaussian3DData_getp_internal_record(el, part);
     if (lumi_record){
         lumi_table       = BeamBeamBiGaussian3DRecordData_getp_lumitable(lumi_record);
-        lumi_table_index =                      LumiTableData_getp__index(lumi_table);
 
     const int at_turn = LocalParticle_get_at_turn(part);
-    /*gpuglmem*/ double* lumi_address = LumiTableData_getp1_luminosity(lumi_table, at_turn);  // double pointer
+    double* lumi_address = LumiTableData_getp1_luminosity(lumi_table, at_turn);  // double pointer
```

Comment:
Should this have been deleted? It leads to an issue: https://github.com/xsuite/xsuite/actions/runs/11494970688/job/31994155254